week08/movie/src/pages/HomePage.jsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import styled from "styled-components";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import Movie from "../components/Movie";
+import { BeatLoader } from "react-spinners";
+
+const Container = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`;
+
+const Loader = styled(BeatLoader)`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+`;
+
+const SortContainer = styled.div`
+    width: 100vw;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+`;
+
+const SortList = styled.div`
+    margin-top: 10px;
+    margin-right: 30px;
+`;
+
+const MovieList = styled.div`
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+`;
+
+const StyledLink = styled(Link)`
+    text-decoration: none;
+    color: inherit;
+`;
+
+const Text = styled.div`
+    font-size: 40px;
+    margin-left: 15px;
+    margin-top: 5px;
+`;
+
+const HomePage = () => {
+    const [loading, setLoading] = useState(true);
+    const [movies, setMovies] = useState([]);
+    const [sort, setSort] = useState("download_count");
+
+    useEffect(() => {
+        async function getMovies() {
+            try {
+                const res = await axios.get(
+                    `https://yts.mx/api/v2/list_movies.json?limit=50&sort_by=${sort}`
+                );
+                setMovies(res.data.data.movies);
+                setLoading(false);
+                console.log(movies);
+            } catch (error) {
+                console.log(error);
+            }
+        }
+        getMovies();
+    }, [sort]);
```

Comment:
2. 변수 `sort`가 업데이트될 때마다, localStorage에서 key가 `SAVED_SORT_KEY`인 데이터의 값을 변수 `sort`의 값으로 업데이트합니다.

```suggestion
        localStorage.setItem(SAVED_SORT_KEY, JSON.stringify(sort));
    }, [sort]);
```